diff --git a/node_modules/three/examples/jsm/loaders/GLTFLoader.js b/node_modules/three/examples/jsm/loaders/GLTFLoader.js
index d8f9b80..8af276a 100644
--- a/node_modules/three/examples/jsm/loaders/GLTFLoader.js
+++ b/node_modules/three/examples/jsm/loaders/GLTFLoader.js
@@ -2453,7 +2453,7 @@ class GLTFParser {
 		let isFirefox = false;
 		let firefoxVersion = - 1;
 
-		if ( typeof navigator !== 'undefined' ) {
+		if ( typeof navigator?.userAgent !== 'undefined' ) {
 
 			isSafari = /^((?!chrome|android).)*safari/i.test( navigator.userAgent ) === true;
 			isFirefox = navigator.userAgent.indexOf( 'Firefox' ) > - 1;
